.travis.yml

```diff
@@ -43,7 +43,8 @@ install:
 
 script:
    - cd test
-   - coverage run --source=jpegenc -m py.test --include-reference
+   - coverage run --source=jpegenc -m py.test --sim=ghdl 
+   - coverage run --source=jpegenc -m py.test --sim=iverilog
```

Comment:
I still want the travis-ci to run the reference design cosimulation `--include-reference` and that will only run with the Verilog simulators.